docs/01-app/03-api-reference/06-cli/next.mdx

```diff
@@ -74,7 +75,8 @@ The following options are available for the `next build` command:
 | ---------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `-h, --help`                       | Show all available options.                                                                                                                                                        |
 | `[directory]`                      | A directory on which to build the application. If not provided, the current directory will be used.                                                                                |
-| `--turbopack`                      | Build using [Turbopack](/docs/app/api-reference/turbopack) (beta). Also available as `--turbo`.                                                                                    |
+| `--turbopack`                      | Build using [Turbopack](/docs/app/api-reference/turbopack). Also available as `--turbo`.                                                                                           |
```

Comment:
Following my question above, we may need to say that this is enabled by default.